rootfs/usr/local/include/toolbox/kops/Makefile.defaults

```diff
@@ -9,7 +9,7 @@ export KOPS_NAME ?= $(CLUSTER_NAME)
 export KOPS_STATE_STORE ?= s3://$(CLUSTER_STATE_BUCKET)
 
 export KOPS_CLOUD ?= aws
-export KOPS_ZONES ?= us-west-2a
+export KOPS_ZONES ?= $(AWS_REGION)a
```

Comment:
Hot sure how whole geodesic workflow was build but as I see in `Makefile.defaults` some variables set by result of external shell calls (just [this](https://github.com/cloudposse/geodesic/blob/master/rootfs/usr/local/include/toolbox/kops/Makefile.defaults#L22) example). 

So, why not use the same way to define first available AZ in region ? something like this:

```
export KOPS_ZONES ?= $(shell  aws ec2 describe-availability-zones  --filters Name=state,Values=available | jq -r '.AvailabilityZones[0].ZoneName')
```

>note: of course `aws cli` and `jq` should be installed into environment where `make` runs